### PR TITLE
Challenge 3 rpetrov

### PR DESCRIPTION
**Pull request by rumenpetrov**
# Pull Request checklist

* [x] My branch follows the naming convention e.g. "challenge-1/asavov"
* [x] I am contributing single HTML file, no folders, no any other external files even icons or images. - [Image CDN](https://placehold.co/)
* [x] My HTML is 100% valid - [HTML Validator](https://validator.w3.org/#validate_by_input)
* [x] My code is semantic (HTML/CSS/JS)
* [x] I preferred native elements than external libraries/packages
* [x] I used meaningful class names
* [x] The elements, properties or APIs I used have sufficient browser support, required by the task I worked on
* [x] My code is performant
* [x] My code follows the general best practices in web development and the style guidelines of this project

### CODE REVIEW

File: public/challenge-contributions/3/rpetrov.html [hunk 1]
```diff
@@ -4,8 +4,211 @@
...
+    }
+    html,
+    body {
+      min-height: 100vh;
+      background-color: #1d1d1d;
+      color: #fff;
+    }
+    .list {
+      display: grid;
+      grid-template-columns: repeat(auto-fit, minmax(min(200px, 100%), 1fr));
```
Comment (sunk0) [inline public/challenge-contributions/3/rpetrov.html:22]: Nice

File: public/challenge-contributions/3/rpetrov.html [hunk 2]
```diff
@@ -4,8 +4,211 @@
...
+      border-color: rgb(255, 255, 255, 0.4);
+      background-color: #000;
+      transition: transform 1s ease, border-color 0.5s ease, box-shadow 0.5s ease;
+      transform-style: preserve-3d;
+      border-radius: var(--radii);
+      box-shadow: 0 0 8px rgba(255, 255, 255, .4);
+    }
+    .card:hover {
+      transform: rotateY(180deg);
+      box-shadow: 0 0 8px rgba(152, 2, 182, 0.6);
```
Comment (sunk0) [inline public/challenge-contributions/3/rpetrov.html:51]: Why the shadow is in the hover state? The shadow moves too?

File: public/challenge-contributions/3/rpetrov.html [hunk 3]
```diff
@@ -4,8 +4,211 @@
...
+    .card__face {
+      box-sizing: border-box;
+      position: absolute;
+      top: 0;
+      left: 0;
+      backface-visibility: hidden;
+      display: flex;
+      height: 100%;
+      width: 100%;
+      overflow: hidden;
```
Comment (sunk0) [inline public/challenge-contributions/3/rpetrov.html:79]: Why do you need to overflow: hidden here? Is backface-visibility: hidden; not enough?

File: public/challenge-contributions/3/rpetrov.html [hunk 4]
```diff
@@ -4,8 +4,211 @@
...
+      align-items: flex-end;
+    }
+    .card__face--front {
+      --pattern-size: 30px;
+      --pattern-color1: #1d1d1d;
+      --pattern-color2: #4e4f51;
+      --pattern-color3: #3c3c3c;
+
+      background:
+        repeating-conic-gradient(from 30deg, #0000 0 120deg, var(--pattern-color3) 0 180deg) calc(.5 * var(--pattern-size)) calc(.5 * var(--pattern-size)*0.577),
```
Comment (sunk0) [inline public/challenge-contributions/3/rpetrov.html:91]: ffs :D

File: public/challenge-contributions/3/rpetrov.html (hunk 2 above)
Comment (rumenpetrov) [reply public/challenge-contributions/3/rpetrov.html:51]: I am changing the color and the intensity of the shadow on hover. The shadow is on the card element (both sides)  and the element is rotating/moves

File: public/challenge-contributions/3/rpetrov.html (hunk 3 above)
Comment (rumenpetrov) [reply public/challenge-contributions/3/rpetrov.html:79]: I need overflow hidden to cut out the rounded corners when I have an image inside, for example.
`backface-visibility: hidden;` is doing something different from `overflow: hidden;`, it removes the back of  an element which is transformed.

**Pull request merged**